### ISSUE TITLE
Point queries should be documented

### ISSUE THREAD

**Issue by fluffysquirrels** [bevyengine/bevy#3170]
### Discussed in https://github.com/bevyengine/bevy/discussions/3169

I came across an issue because I didn't know about point queries (e.g. `Query::get`).  They don't seem to be documented in the book, the unofficial book or the [ecs_guide example](https://github.com/bevyengine/bevy/blob/latest/examples/ecs/ecs_guide.rs).

**Comment by alice-i-cecile** [comment]
Absolutely! They're covered in depth in the ECS chapter of the new book (https://github.com/bevyengine/bevy-website/pull/182), which will be released with 0.6.

**Comment by fluffysquirrels** [comment]
Sounds great!

**Comment by BD103** [comment]
Hi! Looks like this never got closed, but `Query::get()` and friends are documented in the ["Accessing query items" table](https://dev-docs.bevyengine.org/bevy/ecs/prelude/struct.Query.html#accessing-query-items) in `Query`'s docs. Better late than never :)
